### ISSUE TITLE
the app is frozen in catalina

### ISSUE THREAD

**Issue by camiloDD** [quodlibet/quodlibet#3297]
the app is frozen at the moment it goes to the next track, how can I fix it?

**Comment by lazka** [comment]
yeah, that plugin is unstable on all platforms and leads to freezes sadly.
